test/quantization/core/test_backend_config.py

```diff
@@ -142,14 +142,15 @@ def _get_backend_op_config2(self):
 
     def _get_backend_pattern_config_dict1(self):
         return {
-            "pattern": (torch.nn.ReLU, torch.nn.Linear),
+            "pattern": (torch.nn.Linear, torch.nn.ReLU),
             "observation_type": ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT,
             "dtype_configs": [self.dtype_config_dict1, self.dtype_config_dict2],
             "root_module": torch.nn.Linear,
             "qat_module": nnqat.Linear,
             "reference_quantized_module_for_root": nnqr.Linear,
             "fused_module": nni.LinearReLU,
             "fuser_method": self._fuser_method,
+            "use_legacy_pattern_format": False,
```

Comment:
I feel we can rename this to "use_complex_format" or something, so that we don't need new strings in the future when we refactor the complex format?